test/unit/src/math/Vector3.tests.js

```diff
@@ -1006,6 +1006,16 @@ export default QUnit.module( 'Maths', () => {
 
 		} );
 
+		QUnit.test( 'iterable', ( assert ) => {
+
+			var v = new Vector3( 0, 0.5, 1 );
+			var array = [ ...v ];
+			assert.strictEqual( array[ 0 ], 0, 'Vector3 is iterable.' );
+			assert.strictEqual( array[ 1 ], 0.5, 'Vector3 is iterable.' );
+			assert.strictEqual( array[ 2 ], 1, 'Vector3 is iterable.' );
```

Comment:
could you not use this here and the other cases?

```
assert.deepEqual( array, [ 0, 0.5, 1 ] ) 
```

For quaternions either do it like this or keep the long form:

```
var q = new Quaternion( 1, 1, 1 )
var array = [ ...q ]
assert.deepEqual( array.slice( 0, 3 ), [ 1, 1, 1 ] )
assert.strictEqual( array[ 3 ], 3 )
```

Asking this also because I'm curious about preferences regarding this or if there's some related history 👍

Reply:
I just honored the current approach for comparing arrays to keep the code consistent. Feel free to update the unit tests in a separate PR with your suggestion.